### PR TITLE
Goptions.declare_* functions return unit instead of a write_function

### PR DESCRIPTION
**Pull request by SkySkimmer**
Returning a writer insinuates that it is not exactly the same as the
writer which was passed as argument, but that is incorrect.

### CODE REVIEW

**Review by ejgallego — APPROVED** [review]
OK with this, let's see if it breaks anything.

File: plugins/cc/ccalgo.ml
Comment (vbgl) [inline plugins/cc/ccalgo.ml]: Missing whitespace?

File: plugins/firstorder/g_ground.mlg
Comment (vbgl) [inline plugins/firstorder/g_ground.mlg]: Whitespaces.

File: plugins/ssr/ssrfwd.ml
Comment (vbgl) [inline plugins/ssr/ssrfwd.ml]: `let () =` instead of `let _ =`

File: pretyping/evarconv.ml
Comment (vbgl) [inline pretyping/evarconv.ml]: `let ()`

**Review by vbgl — APPROVED** [review]

**Comment by ejgallego** [comment]
This seem ready to go; maintainers of this file are @silene and @gares tho @herbelin has also expressed interest in `GOptions`. I can also assign it if the maintainers are busy.

**Comment by ejgallego** [comment]
Will merge tomorrow if maintainers remain silent.

**Comment by herbelin** [comment]
Good to me.

**Pull request merged**
